tests/python/unittest/test_metric.py

```diff
@@ -122,6 +123,54 @@ def test_f1():
     np.testing.assert_almost_equal(microF1.get()[1], fscore_total)
     np.testing.assert_almost_equal(macroF1.get()[1], (fscore1 + fscore2) / 2.)
 
+def test_mcc():
```

Comment:
could you add a consistency test with Matthew's Correlation Coefficient in numpy?

Reply:
never mind. I didn't realize that numpy doesn't have it.